src/core/randomizer.js

```diff
@@ -15,13 +15,13 @@ function buildRandomizer() {
       let newMessagesState;
 
       if (messagesState.length !== 0) {
-        newMessage = messagesState.at(index);
```

Comment:
Was this replaces automatically by IDE? Is there any concerns using `.at(index)` over `[index]`. I'm okay with `[index]` too, just wanted to know you opinion 😊

Reply:
No not IDE , it was intentional, I thought it could be more readable this way 😅

Reply:
Fine! Let's keep it then 😆